roofit/roofit/src/LinearCombination.cxx

```diff
@@ -0,0 +1,151 @@
+#include "LinearCombination.h"
+#include "Floats.h"
+
+#include "Math/Util.h"
+
+ClassImp(LinearCombination);
+
+namespace {
+template <class T> inline void assign(SuperFloat &var, const T &val) {
+#ifdef USE_UBLAS
+  var.assign(val);
+#else
+  var = val;
+#endif
+}
+inline double convertToDouble(const SuperFloat &var) {
+#ifdef USE_UBLAS
+  return var.convert_to<double>();
+#else
+  return var;
+#endif
+}
+} // namespace
+
```

Comment:
```suggestion
```
Since we use do the preprocessor `ifdef` now in `evaluate()`, these functions should be removed to not be reported as unused functions if `USE_UBLAS` is not set.